keytar_windows.go

```diff
@@ -118,24 +60,12 @@ func (*keychainWindows) DeletePassword(service, account string) error {
 		return ErrInvalidValue
 	}
 
-	// Compute target item name
-	target := targetFormat(service, account)
-
-	// Convert the target name.  See note in AddPassword.
-	targetUTF16Ptr, err := syscall.UTF16PtrFromString(target)
+	cred, err := wincred.GetGenericCredential(targetFormat(service, account))
 	if err != nil {
-		return ErrInvalidValue
-	}
-	// NOTE: For some reason they use LPCWSTR here, as opposed to LPWSTR in the
-	// CREDENTIALW struct
-	targetUTF16 := C.LPCWSTR(unsafe.Pointer(targetUTF16Ptr))
-
-	// Delete the credential
-	if C.CredDeleteW(targetUTF16, C.CRED_TYPE_GENERIC, 0) != C.TRUE {
-		return ErrUnknown
+		return err
 	}
+	cred.Delete()
 
-	// All done
 	return nil
 }
 
```

Comment:
(See below) This one is my fault, but if you make the two previous changes, can you change the comment in the `init()` function to "Register the Windows keychain implementation"?

Reply:
Done